crates/version/build.rs

```diff
@@ -0,0 +1,36 @@
+use std::process::Command;
+
+fn main() {
+    let git_describe = Command::new("git")
+        .args([
+            "describe",
+            "--always",
+            "--dirty=-modified",
+            "--tags",
+            "--match",
+            "[0-9]*",
+        ])
+        .output()
+        .ok()
+        .and_then(|o| String::from_utf8(o.stdout).ok())
+        .unwrap_or_else(|| "unknown".to_string());
```

Comment:
let's get better error handling here

consider breaking up the main function into

`fn main() {}` and `fn try_main() -> eyre::Result<()> {}`

within main, you can then match on the result of try_main, printing the error and exiting with code 1 if there is an error

for reference - https://github.com/near/nearcore/blob/221eb4ff26fcf3b964a7a9f875cfa14fa5277499/neard/build.rs#L119-L124